gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImplCreateTest.java

```diff
@@ -206,13 +306,35 @@ public void testCreateObjectApiInterruptedException() throws Exception {
     assertWithMessage("Neither thread started.")
         .that(writeStartedLatch.await(5000, TimeUnit.MILLISECONDS))
         .isTrue();
+    // callForClose will be waiting on write(waiting forever) to finish. Interrupt it.
     write.cancel(/* interrupt= */ true);
     assertWithMessage("Failed to wait for tasks to get interrupted.")
         .that(threadsDoneLatch.await(5000, TimeUnit.MILLISECONDS))
         .isTrue();
   }
 
-  private GoogleCloudStorage getCloudStorageImpl(HttpTransport transport) throws IOException {
-    return tesStorageClientImpl ? mockedGcsClientImpl(transport) : mockedGcsImpl(transport);
+  static StorageObject newStorageObject(String bucketName, String objectName) {
```

Comment:
This probably should be part of test helper class. Maybe we have already one?

Reply:
Moved.

Reply:
Seems like not moved?

Reply:
got left, refactored now.